br/br-snapshot-guide.md

```diff
@@ -0,0 +1,180 @@
+
+To restore a database to a cluster, run the `br restore db` command. The following example restores the `test` database from the backup data to the target cluster:
+
+```shell
+tiup br restore db \
+--pd "${PD_IP}:2379" \
+--db "test" \
+--storage "s3://backup-101/snapshot-202209081330?access_key=${access_key}&secret_access_key=${secret_access_key}"
+```
+
+In the preceding command, `--db` specifies the name of the database to be restored.
+
+**Restore a table**
+
+To restore a single table to a cluster, run the `br restore table` command. The following example restores the `test.usertable` table from the backup data to the target cluster:
+
+```shell
+tiup br restore table --pd "${PD_IP}:2379" \
+--db "test" \
+--table "usertable" \
+--storage "s3://backup-101/snapshot-202209081330?access_key=${access_key}&secret_access_key=${secret_access_key}"
+```
+
+In the preceding command, `--db` specifies the name of the database to be restored, and `--table` specifies the name of the table to be restored.
+
+**Restore multiple tables with table filter**
+
+To restore multiple tables with more complex filter rules, run the `br restore full` command and specify the [table filters](/table-filter.md) with `--filter` or `-f`. The following example restores tables that match the `db*.tbl*` filter rule from the backup data to the target cluster:
+
+```shell
+tiup br restore full \
+--pd "${PD_IP}:2379" \
+--filter 'db*.tbl*' \
+--storage "s3://backup-101/snapshot-202209081330?access_key=${access_key}&secret_access_key=${secret_access_key}"
+```
+
+### Restore tables in the `mysql` schema
+
+Starting from `br` v5.1.0, when you back up snapshots, `br` backs up the **system tables** in the `mysql` schema and does not restore them by default. Starting from `br` v6.2.0, `br` restores **data in some system tables** if you configure `--with-sys-table`.
+
+**The `br` command-line tool can restore data in the following system tables:**
+
+```
++----------------------------------+
+| mysql.columns_priv               |
+| mysql.db                         |
+| mysql.default_roles              |
+| mysql.global_grants              |
+| mysql.global_priv                |
+| mysql.role_edges                 |
+| mysql.tables_priv                |
+| mysql.user                       |
++----------------------------------+
+```
+
+**The `br` command-line tool does not restore the following system tables:**
+
+- Statistics tables (`mysql.stats_*`)
+- System variable tables (`mysql.tidb` and `mysql.global_variables`)
+- [Other system tables](https://github.com/pingcap/tidb/blob/master/br/pkg/restore/systable_restore.go#L31)
+
+When you restore data related to system privilege, note the following:
+
+- `br` does not restore user data with `user` as `cloud_admin` and `host` as `'%'`. This user is reserved for TiDB Cloud. Do not create a user or role named `cloud_admin` in your cluster, because the user privileges related to `cloud_admin` cannot be restored correctly.
```

Comment:
```suggestion
- `br` does not restore user data with `user` as `cloud_admin` and `host` as `'%'`. This user is reserved for TiDB Cloud. Stop creating a user or role named `cloud_admin` in your cluster, because the user privileges related to `cloud_admin` cannot be restored correctly.
```